### DISCUSSION TITLE
Advantages over consola?

### DISCUSSION THREAD

**Discussion by radiorambo** [adzejs/adze#184] [Q&A] [answered]
I respect you for all the time you invested to build this library. 

So I long ago I was in search of a single logging library that I can use both frontend and backend and ended up with `consola`. 

So I want to know what advantages does adze  offer over consola?

Thanks

**Comment by AJStacy** [comment, marked as answer]
I don't have a lot of experience with consola, but Adze is more than just a universal logging library. The fact that it is universal and works in any JS/TS runtime is one of its primary features, but it also has a much larger array of functionality than it appears consola has (middleware, listeners, MDC, chaining API's with meta data markup, customizability, machine-readable logs, etc).

Please refer to this page for any overview of the feature set of Adze: https://adzejs.com/getting-started/introduction.html#why-should-i-use-adze
